meshdb/meshdb.go

```diff
@@ -16,6 +16,13 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	// The default miniHeaderRetentionLimit used by Mesh. This default only gets overwritten in tests.
+	defaultMiniHeaderRetentionLimit = 20
+	// The maximum MiniHeaders to query per page when deleting MiniHeaders
+	miniHeadersMaxPerPage = 5000
```

Comment:
As mentioned yesterday offline, we should change this to `1000`. The memory usage is quite a bit lower and the execution time is almost identical (and actually a bit quicker on average).

Reply:
@fabioberger you resolved the conversation but this is still set to `5000`. Can you update it to `1000`?